backend/src/schema/resolvers/searches.spec.js

```diff
@@ -416,6 +431,128 @@ und hinter tausend Stäben keine Welt.`,
         })
       })
 
+      describe('adding a user and a hashtag with a name that is content of a post', () => {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-06-22T13:14:13Z" title="Monday, June 22nd 2020, 3:14:13 pm +02:00">Jun 22, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -416,6 +431,128 @@ und hinter tausend Stäben keine Welt.`,
         })
       })
 
+      describe('adding a user and a hashtag with a name that is content of a post', () => {
```


Hey, thanks for all the new tests here! 😍